src/libraries/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Ssl.cs

```diff
@@ -119,10 +119,10 @@ internal enum PAL_TlsIo
         private static extern int AppleCryptoNative_SslSetAcceptClientCert(SafeSslHandle sslHandle);
 
         [DllImport(Interop.Libraries.AppleCryptoNative, EntryPoint = "AppleCryptoNative_SslSetIoCallbacks")]
-        internal static extern int SslSetIoCallbacks(
+        internal static extern unsafe int SslSetIoCallbacks(
             SafeSslHandle sslHandle,
-            SSLReadFunc readCallback,
-            SSLWriteFunc writeCallback);
+            delegate* unmanaged<void*, byte*, void**> readCallback,
+            delegate* unmanaged<void*, byte*, void**> writeCallback);
```

Comment:
```suggestion
            delegate* unmanaged<void*, byte*, void**, int> readCallback,
            delegate* unmanaged<void*, byte*, void**, int> writeCallback);
```
This should fix the build break